### DISCUSSION TITLE
Delete button in the Layer Manager

### DISCUSSION THREAD

**Discussion by Duvan1** [GrapesJS/grapesjs#5041] [General]
Hello, first of all thank you for providing us with this great tool.
My question is about the actions that can be done in the layer manager, I know that by default you can move a layer and hide it, but I wanted to know if by default it is possible to add an icon that allows you to delete the layer directly from the layer manager.
Thank you.
![layer-manager](https://user-images.githubusercontent.com/28686863/230985611-3754eb35-762a-4a43-8a3d-2caeba21234f.png)

**Comment by artf** [comment]
You can create your own Layer Manager UI https://grapesjs.com/docs/modules/Layers.html#customization
